### PR TITLE
add labels in sig list

### LINKED ISSUE #1172: SIG list doesn't help with /sig command
When the k8s-merge-bot tells me to add a /sig label ([example](https://github.com/kubernetes/kubernetes/issues/53683#issuecomment-335624804)), It redirects me to this list: https://github.com/kubernetes/community/blob/master/sig-list.md

But this list is all human-readable names e.g. "Contributor Experience" but I can't use this in /sig command. It's unclear to find out which one is the correct value:
- /sig Contributor Experience
- /sig "Contributor Experience"
- /sig contributor-experience

I had to visit https://github.com/kubernetes/kubernetes/labels to find out which values are valid.

### PR DESCRIPTION
**Pull request by yuexiao-wang**
Signed-off-by: yuexiao-wang <wang.yuexiao@zte.com.cn>

fix #1172

It's unclear to find out which one is the correct value in sig list when use /sig command.
Therefore, I suggest that add labels in list.

### CODE REVIEW

**Comment by castrojo** [comment]
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/community/1173)

**Comment by yuexiao-wang** [comment]
Got it and I will do it.
Thanks

**Comment by yuexiao-wang** [comment]
@cblecker PTAL

File: sig-list.md [hunk 1]
```diff
@@ -20,36 +20,36 @@ When the need arises, a [new SIG can be created](sig-creation-procedure.md)
```
Comment (xiangpengzhao) [inline sig-list.md]: don't change the file mode :)

File: sigs.yaml [hunk 2]
```diff
@@ -485,6 +504,7 @@ sigs:
       source project is very new and largely unscoped for a project as large
       as Kubernetes; we are learning too and we are excited to learn how we
       can best serve the community of users and contributors.
+    label: product-management
```
Comment (xiangpengzhao) [inline sigs.yaml]: this LG, but please note that we don't have this label in K/K yet.

File: generator/sig_readme.tmpl [hunk 3]
```diff
@@ -10,6 +10,9 @@
 Meeting notes and Agenda can be found [here]({{.MeetingArchiveURL}}).
 Meeting recordings can be found [here]({{.MeetingRecordingsURL}}).
 
+## Label
```
Comment (cblecker) [inline generator/sig_readme.tmpl]: I'm not sure adding this is needed in the sig readme.. doesn't add a lot of information.

Comment (yuexiao-wang) [reply generator/sig_readme.tmpl]: Please see the issue  #1172 by @ahmetb.  Users find sig list doesn't help /sig command  and have a confuse which are valid value in commond as following:

When the k8s-merge-bot tells me to add a /sig label (example), It redirects me to this list: https://github.com/kubernetes/community/blob/master/sig-list.md

But this list is all human-readable names e.g. "Contributor Experience" but I can't use this in /sig command. It's unclear to find out which one is the correct value:
I had to visit https://github.com/kubernetes/kubernetes/labels to find out which values are valid.

Comment (xiangpengzhao) [reply generator/sig_readme.tmpl]: I think @cblecker meant that it's not so necessarily to add label to the README file of each sig.
Just add a `Label` column to sig-list.md (as you already did in this PR) to indicate the sig labels would be enough.

Comment (cblecker) [reply generator/sig_readme.tmpl]: Yes, thanks for clarifying @xiangpengzhao. That's what I meant. Putting the label into the sig-list.md template is good, but I don't think we need a separate section in every README.md with just the label.

Comment (yuexiao-wang) [reply generator/sig_readme.tmpl]: @cblecker What do I do for it?  I have to add label into sig.yml and add lable field into Group in app.go. Then exectue the "make" command and all README.md were added label.

Comment (yuexiao-wang) [reply generator/sig_readme.tmpl]: OK, I got it and update

**Comment by yuexiao-wang** [comment]
@cblecker @xiangpengzhao Done

**Comment by yuexiao-wang** [comment]
@cblecker @xiangpengzhao All labels are added in sig-list.md. PTAL

**Comment by xiangpengzhao** [comment]
/lgtm
Note that we already support `wg/*` label (https://github.com/kubernetes/test-infra/pull/4358). But there are still no such labels in our repos. There can be a followup PR once the `wg` are added.
@bgrant0607 are we ready to add `wg/*` ?

**Pull request merged**